sensor/kubernetes/listener/resources/complianceoperator/dispatchers/complianceoperatorscans.go

```diff
@@ -33,8 +35,63 @@ func (c *ScanDispatcher) ProcessEvent(obj, _ interface{}, action central.Resourc
 		return nil
 	}
 
+	// useful for the deduping from sensor.
+	uid := string(complianceScan.UID)
+
+	// We probably could have gotten away with re-using the storage proto here for the time being.
+	// But we have a new field coming on for profiles and using the storage object even in an internal api
+	// is a bad practice, so we will make that split now.  V1 and V2 compliance will both need to work for a period
+	// of time.  However, we should not need to send the same profile twice, the pipeline can convert the V2 sensor message
+	// so V1 and V2 objects can both be stored.
+	// TODO(ROX-21152) switch to capability
```

Comment:
Depending on which PR gets merged first this one or #8889 as to if I switch to capability in this PR or that one.